### PR TITLE
Add Logitech

### PR DESCRIPTION
**Pull request by niekvb**

### CODE REVIEW

File: entries/l/logitech.com.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "logi.com"
+    ],
+    "img": "logitech.com.png",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "keywords": [
+      "other",
+      "gaming"
```
Comment (Carlgo11) [inline entries/l/logitech.com.json]: The __gaming__ category is for games and game-related services such as Steam and Origin whilst the __other__ category is for things that can't be categorized.
I suggest you use the __retail__ category instead and Logitech is primarily in the business of selling consumer products.
```suggestion
      "retail"
```

File: entries/l/logitech.com.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "Logitech": {
+    "domain": "logitech.com",
+    "additional-domains": [
+      "astrogaming.com",
+      "logi.com"
+    ],
+    "img": "logitech.com.png",
```
Comment (Carlgo11) [inline entries/l/logitech.com.json]: The PNG you provided seem to be for the Logitech G brand. Here's an SVG of the official Logitech logo:

```SVG+XML
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 45.87 14"><path fill="#00b8fc" d="M13.95 8.46a1.95 1.95 0 0 1-1.81-1.22 1.98 1.98 0 0 1 1.43-2.69 1.95 1.95 0 0 1 2.01.84 1.98 1.98 0 0 1 .19 1.86 1.95 1.95 0 0 1-1.82 1.21zm3.6-4.02V3.08h-2.38a3.7 3.7 0 0 0-1.22-.21 3.6 3.6 0 0 0-3.67 3.62 3.6 3.6 0 0 0 3.67 3.62 3.6 3.6 0 0 0 3.11-5.5l.49-.17zM6.03 8.46A1.95 1.95 0 0 1 4.2 7.24a1.98 1.98 0 0 1 1.43-2.69 1.95 1.95 0 0 1 2.02.84 1.98 1.98 0 0 1-1.63 3.07zm0-5.6A3.6 3.6 0 0 0 2.36 6.5a3.6 3.6 0 0 0 3.67 3.62A3.6 3.6 0 0 0 9.7 6.5a3.6 3.6 0 0 0-3.67-3.62zM0 9.92h1.7V.2H0Zm18.3 0h1.69V3.08h-1.7Zm-4.35 2.43a1.92 1.92 0 0 1-1.8-1.2c-.1-.25-.15-.5-.15-.76h-1.72A3.6 3.6 0 0 0 13.95 14a3.6 3.6 0 0 0 3.67-3.62H15.9a1.95 1.95 0 0 1-1.95 1.96zm12.28-6.56c.22-.9.91-1.4 1.8-1.4s1.55.48 1.76 1.4zm1.84-2.91a3.55 3.55 0 0 0-3.59 3.66 3.43 3.43 0 0 0 3.62 3.58c1.22 0 2.3-.45 3.1-1.3l-1.14-1.13c-.56.53-1.16.92-1.92.92-.93 0-1.63-.54-1.87-1.43h5.28v-.41c0-2.34-1.43-3.9-3.48-3.9zm15 0a2.73 2.73 0 0 0-2.04.86V.19h-1.7V9.9h1.7V6.2c0-.99.62-1.67 1.65-1.67.97 0 1.5.6 1.5 1.67V9.9h1.7V5.9c0-1.9-1.18-3.02-2.8-3.02zm-7.3 1.65c.7 0 1.3.37 1.74.9l1.18-1.18a3.68 3.68 0 0 0-2.9-1.38 3.6 3.6 0 0 0-3.67 3.63 3.6 3.6 0 0 0 3.66 3.62 3.66 3.66 0 0 0 2.91-1.37l-1.17-1.17c-.44.5-1.04.89-1.77.89a1.91 1.91 0 0 1-1.92-1.97c0-1.12.84-1.98 1.94-1.98zM23.32.97h-1.7v2.1h-.91V4.6h.91V9.9h1.7V4.6h.93V3.08h-.93ZM19.13 0a1.1 1.1 0 0 0-.8.33c-.2.21-.32.5-.31.8a1.14 1.14 0 0 0 1.12 1.14 1.12 1.12 0 0 0 1.13-1.14A1.12 1.12 0 0 0 19.13 0Z"/></svg>
```

**Review by Carlgo11 — COMMENTED** [review]
I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
screenshot

File: entries/l/logitech.com.json (hunk 2 above)
Comment (niekvb) [reply entries/l/logitech.com.json]: I modified it to match their favicon

File: entries/l/logitech.com.json (hunk 1 above)
Comment (niekvb) [reply entries/l/logitech.com.json]: Sure

**Comment by niekvb** [comment]
> I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?
> 
> If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.

I can't find any useful documentation from Logitech.

Taking screenshots probably won't help much as the site redirects me to the Dutch version.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**